aivs-monitor编译尝试报错缺少curl.h

交叉编译环境使用下面的：
open-lx01/tree/main/src/cross-build-env
make方式准备的交叉编译环境 或者 build好的image都试过

根据：
open-lx01/tree/main/src/apps/aivs-monitor
中的说明
make build-all
报错：
make: *** No rule to make target 'build-all'. Stop.

观察
open-lx01/blob/main/src/cross-build-env/Makefile
发现问题：
根据提供的Makefile内容，无法直接使用 make build-all，因为该Makefile中并没有定义名为 build-all 的目标

发现makefile中存在：
build-with-docker:
	docker run --rm -v $(TOP_DIR):/build -w /build $(BUILD_IMAGE) sh -c "cd $(MAKEFILE_DIR) && make build"

故尝试执行：
make build-with-docker

依然报错提示没有curl.h

--- a/.github/workflows/aivs-monitor.yml
+++ b/.github/workflows/aivs-monitor.yml
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
 

--- a/.github/workflows/pkg-building.yml
+++ b/.github/workflows/pkg-building.yml
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   building:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
 

--- a/.github/workflows/server.yml
+++ b/.github/workflows/server.yml
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
 

--- a/src/apps/aivs-monitor/README.md
+++ b/src/apps/aivs-monitor/README.md
@@ -5,8 +5,13 @@
 需要依赖交叉编译环境, [参考](../../cross-build-env/)
 
 ```sh
-make build-all
+# 如果已经 build 了所有依赖, 只是想重新编译 aivs-monitor, 那么在当前目录 make 即可
+make
+
+# 如果 build 报错提示找不到 libcurl 等依赖, 那么在上层文件夹, 也就是 app 目录 make 等待所有依赖 build 完成即可
 ```
+
+注: make 出来的是目标 arm 平台的 binary, 需要拷贝到音箱上才能执行 
 
 TODO
 

--- a/src/server/requirements.txt
+++ b/src/server/requirements.txt
@@ -8,4 +8,6 @@ scipy~=1.11.4
 requests~=2.31.0
 pydantic~=2.5.3
 openai~=1.10.0
+# https://community.openai.com/t/error-with-openai-1-56-0-client-init-got-an-unexpected-keyword-argument-proxies/1040332/2
+httpx==0.27.2
 peewee

